docs/cmd/kn_container_add.md

```diff
@@ -0,0 +1,61 @@
+## kn container add
+
+Add a container
+
+```
+kn container add NAME
+```
+
+### Examples
+
+```
+
+  The 'container add' represents utility command that prints YAML container spec to standard output. It's useful for
+  multi-container use cases to create definition with help of standard 'kn' option flags. The command can be chained through
+  Unix pipes to create multiple containers at once.
+
+  # Add a container 'sidecar' from image 'docker.io/example/sidecar'
+  kn container add sidecar --image docker.io/example/sidecar
+
+  # Add command can be chained by standard Unix pipe symbol '|'
+  kn container add sidecar --image docker.io/example/sidecar | \
+  kn container add second --image docker.io/example/sidecar:second
+```
```

Comment:
I would add here also the final `kn service create|update|apply` command, so that this example is kind of self-contained